### PR TITLE
the greater good

### PR DESCRIPTION
**Pull request by benghaziboy**
making this an abstract baseclass, but left in the main model for the time being.

@mparent61 @MichaelButkovic

### CODE REVIEW

**Comment by benghaziboy** [comment]
one thing that I noticed will be painful about the json fields, is that changing a serializer name will require some manual data migrations

File: tests/infographics/test_models.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def test_create_success(self):
             infographic_type=InfographicType.LIST,
             data={
                 "is_numbered": True,
-                "items": [{
+                "objects": [{
```
Comment (mparent61) [inline tests/infographics/test_models.py]: wouldn't these be called "entries"? It's also a bit less generic than "objects".

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
 
 ## Development
 
+## Version 3.1.0
+
+- Added `AbstractInfographic` model to refer to all infographic specific attributes.
+- Rename Infographic.data,items to Infographic.data.objects due to
```
Comment (mparent61) [inline CHANGELOG.md]: entries

**Comment by mparent61** [comment]
LGTM

**Pull request merged**